### PR TITLE
Update translations 2.x backport

### PR DESCRIPTION
**Pull request by dhmlau**
### Description
Update translations 2.x backport

connected to https://github.com/strongloop-internal/scrum-apex/issues/149

### CODE REVIEW

**Comment by dhmlau** [comment]
@slnode test please

**Comment by dhmlau** [comment]
@slnode  test please

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# LoopBack Boot
+# LoopBack Boot
```
Comment (bajtos) [inline README.md]: Trailing white-space, please remove.

Comment (dhmlau) [inline README.md]: Thanks @bajtos .  Was trying to force CI to pick up new changes in `gateway-director-testapp` to get it to pass.  Will get you review this PR again when it's ready.

**Comment by dhmlau** [comment]
@bajtos , PTAL. thanks!

**Pull request merged**